common/djangoapps/status/migrations/0001_initial.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from __future__ import absolute_import
```

Comment:
Could you please group together the `__future__` imports, I believe running isort should do the trick.